pokemon_load_data.py

```diff
@@ -33,11 +34,9 @@ class Pokemon(Base):
 # Base.metadata.drop_all(bind=engine)
 Base.metadata.create_all(bind=engine)
 
-
 # Load JSON Data
 app = FastAPI()
 
-
 @app.on_event("startup")
```

Comment:
I suggest this route as `/pokemon/load`
also include this route with other CRED routes of Pokemon, instead of a separate file